awswrangler/timestream.py

```diff
@@ -625,12 +633,12 @@ def list_databases(
 
 
     """
-    client: boto3.client = _utils.client(service_name="timestream-write", session=boto3_session)
+    client = _utils.client(service_name="timestream-write", session=boto3_session)
 
-    response: Dict[str, Any] = client.list_databases()
+    response = client.list_databases()
     dbs: List[str] = [db["DatabaseName"] for db in response["Databases"]]
-    while "nextToken" in response:
-        response = client.list_databases(nextToken=response["nextToken"])
```

Comment:
This was an actual bug fix. The error was surfaced by MyPy thanks to these stubs.